### PR TITLE
command options: Add dumpgenesis command

### PR DESCRIPTION
**Pull request by hyochan-brown**
## Proposed changes
Enhance ethereum compability [(PR)](https://github.com/ethereum/go-ethereum/pull/20191)
- Add dumpgenesis command.  (Dumps genesis block JSON configuration to stdout)
```
ken dumpgenesis --baobab
{"config":{"chainId":1001,"istanbul": ...}, "timestamp":"0x5d132648","extraData":"...","governanceData":...,"blockScore":...,"alloc":...,"number":"0x0","gasUsed":"0x0","parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000"}
```

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/utils/cmd.go [hunk 1]
```diff
@@ -42,6 +43,26 @@ const (
 
 var logger = log.NewModuleLogger(log.CMDUtils)
 
+// Fatalf formats a message to standard error and exits the program.
+// The message is also printed to standard output if standard error
+// is redirected to a different file.
+func Fatalf(format string, args ...interface{}) {
+	w := io.MultiWriter(os.Stdout, os.Stderr)
+	if runtime.GOOS == "windows" {
```
Comment (hyochan-brown) [inline cmd/utils/cmd.go:51]: I have a question.
Is klaytn going to support Windows?
If not, I'll remove this code.

File: cmd/utils/nodecmd/chaincmd.go [hunk 2]
```diff
@@ -164,6 +178,17 @@ func initGenesis(ctx *cli.Context) error {
 	return nil
 }
 
+func dumpGenesis(ctx *cli.Context) error {
+	genesis := utils.MakeGenesis(ctx)
+	if genesis == nil {
+		genesis = blockchain.DefaultGenesisBlock()
+	}
+	if err := json.NewEncoder(os.Stdout).Encode(genesis); err != nil {
+		utils.Fatalf("could not encode genesis")
```
Comment (jeongkyun-oh) [inline cmd/utils/nodecmd/chaincmd.go:187]: How about using `logger.Crit` instead of `utils.Fatalf`?

Comment (hyochan-brown) [reply cmd/utils/nodecmd/chaincmd.go]: I reflected it.
Sorry for late.

**Review by aidan-kwon — DISMISSED** [review]

File: cmd/utils/cmd.go (hunk 1 above)
Comment (jeongkyun-oh) [reply cmd/utils/cmd.go]: I guess we do not need to consider this anymore

**Review by jeongkyun-oh — APPROVED** [review]

**Review by winnie-byun — DISMISSED** [review]

File: cmd/utils/flags.go [hunk 3]
```diff
@@ -1594,6 +1594,17 @@ func SetKlayConfig(ctx *cli.Context, stack *node.Node, cfg *cn.Config) {
 	setTxResendConfig(ctx, cfg)
 }
 
+func MakeGenesis(ctx *cli.Context) *blockchain.Genesis {
+	var genesis blockchain.Genesis
```
Comment (yoomee1313) [inline cmd/utils/flags.go:1598]: @hyochan-brown Can I ask why you didn't declare `genesis` as a pointer?

**Comment by aidan-kwon** [comment]
@hyochan-brown Please merge this PR if you finish all conversations above.

File: cmd/utils/flags.go (hunk 3 above)
Comment (hyochan-brown) [reply cmd/utils/flags.go]: It's my mistake...
Thank you so much.
Not using pointer will skip the nil check and nothing came out in command.

**Review by yoomee1313 — APPROVED** [review]

**Review by winnie-byun — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Pull request merged**